src/components/Firestore/DocumentEditor/index.tsx

```diff
@@ -302,15 +325,20 @@ const FieldEditor: React.FC<{
 const ChildTypeSelect: React.FC<{
   uuid: number;
   isRtdb: boolean;
-}> = ({ uuid, isRtdb }) => {
+  blacklist?: FieldType[];
```

Comment:
Consider adding a `allowedTypes` prop to replace both `isRtdb` and `blacklist`. 
(Alternatively but less preferred, rename to `disabledTypes` for clarity.)

Reply:
yeah, slight pref for a white list and remove rtdb, vs blacklist ..

Reply:
Oh, I see how it's used. blacklist is more lightweight.. carry on :)

Reply:
But I do think it's worth combining blacklist and isRtdb

Reply:
Ack, updated to allowedTypes and merged the two props.